FMB/FiniteModelMultiSorted.cpp

```diff
@@ -792,7 +862,327 @@ void FiniteModelMultiSorted::eliminateSortFunctionsAndPredicates(const Stack<uns
   }
 }
 
-bool FiniteModelMultiSorted::evaluate(Unit* unit)
+void FiniteModelMultiSorted::restoreEliminatedFunDef(Problem::FunDef* fd)
```

Comment:
In the interest of time I'm going to assume you know what you're doing here. Do we have to do this in duplicate for mono- and multi-sorted logic?

Reply:
I don't maintain `FiniteModel`(non-multisorted) and I don't think we have to (currently, all model printing goes through `FiniteModelMultiSorted`). Feel free to remove it!